src/conf_mode/protocols_isis.py

```diff
@@ -203,6 +203,37 @@ def verify(isis):
         if list(set(global_range) & set(local_range)):
             raise ConfigError(f'Segment-Routing Global Block ({g_low_label_value}/{g_high_label_value}) '\
                               f'conflicts with Local Block ({l_low_label_value}/{l_high_label_value})!')
+        
+    # Check for a blank or invalid value per prefix
+    if dict_search('segment_routing.prefix', isis):
+        for prefix in isis['segment_routing']['prefix']:
+            if 'absolute' in isis['segment_routing']['prefix'][prefix]:
+                try:
+                    segment_label_value = int(isis['segment_routing']['prefix'][prefix]['absolute']['value'])
+                    if not 16 <= segment_label_value <= 1048576:
```

Comment:
This is already handled by the XML definition and no additional tests are required.

```xml
            <leafNode name="value">
              <properties>
                <help>Specify the absolute value of prefix segment/label ID</help>
                <valueHelp>
                  <format>u32:16-1048575</format>
                    <description>The absolute segment/label ID value</description>
                </valueHelp>
                <constraint>
                  <validator name="numeric" argument="--range 16-1048575"/>
                </constraint>
              </properties>
            </leafNode>
```

Reply:
Sure, yes, I did know that they were there but I figure maybe we might need two places to keep it from happening? Or should we just keep it in the XML definitions only?

Should I comment it out, or just remove completely.

Reply:
Please remove completely - no need to keep unused code that does not belong there.

Reply:
Will do sir. Thank you.